about.html

```diff
@@ -0,0 +1,190 @@
+      <section class="banner-section banner-section-about-page">
+        <div class="background-image background-image-about-page"></div>
+        <div class="banner-text-content banner-text-content-about">
+          <h1 class="mb-3 text-center">Hello Developers!</h1>
+          <h2 class="fw-bold text-center">
+            ReviewMe, GET INSTANT HELP, <br />
+            MEET PROFESSIONALS AND <br />
+            MOVE 5x FASTER
+          </h2>
+          <div class="bg-white site-description-div">
+            <p class="p-3 text-center">
+              Find qualified code reviewers, get feedback on your code, and
+              improve the quality of your code. Learn new skills, and give
+              feedback to the software development community.
+            </p>
+          </div>
+          <div class="text-center">
+            <p class="mt-3">
+              Please Contact us per Email for any further questions about
+              ReviewMe! <br />
+              <span class="fw-bold text-decoration-underline"
+                >brunokambere@gmail.com</span
+              >
+            </p>
+          </div>
+        </div>
+      </section>
+
+      <section class="reviewers-section">
+        <div class="content-title-service mt-5">
+          <h3 class="text-center">ReviewMe Logo</h3>
+          <div class="reviewers-line"><hr /></div>
+          <div class="text-center">
+            <p>
+              ReviewMe Logo was decided through the logo competition from 10
+              June to 15.
+            </p>
+          </div>
+
+          <div class="about-logo text-center">
+            <img
+              src="assets/images/reviewme-low-resolution-logo-color-on-transparent-background.png"
+              width="150px"
+              height="150px"
+              alt=""
+            />
+          </div>
+        </div>
+      </section>
+      <div class="about-big-line"><hr /></div>
+      <section class="hackathon-about-section text-center">
+        <h4>See the past ReviewMe Hackathon</h4>
+        <div class="service-line"><hr /></div>
+        <p>
+          Take a look at the last two Hackathons which took place in Goma and
+          Kinshasa
+        </p>
+        <div class="hackathon-about-section-wrapper">
+          <div class="hackathon-about-section-image-1">
+            <div class="hackathon-background-image"></div>
+            <div class="hackathon-background-color"></div>
+            <div class="hackathon-description">
+              <h4>2023</h4>
+              <p>ReviewMe Hackathon in Kinshasa</p>
+            </div>
+          </div>
+          <div class="hackathon-about-section-image-2">
+            <div class="hackathon-background-image"></div>
+            <div class="hackathon-background-color"></div>
+            <div class="hackathon-description">
+              <h4>2023</h4>
+              <p>ReviewMe Hackathon in Goma</p>
+            </div>
+          </div>
+        </div>
+      </section>
```

Comment:
- [ ] Kindly make sure to follow the design guidelines as much as possible, for this section, kindly address the following:-

    - You need to use the proper font, and font size to match the actual design.
    -  You need to add the proper paddings, spacings, and margins to adjust the layout.

| Your design | Actual design |
| ------------- | ------------- |
| ![image](https://github.com/kambereBr/ReviewMe/assets/102612396/6617cbc4-80f0-41ea-99b5-1320cf9d2e56) | ![image](https://github.com/kambereBr/ReviewMe/assets/102612396/052bcc3a-c297-4b39-9a39-6445a5d0c0da) |